src/datafusion/utils.rs

```diff
@@ -106,10 +106,13 @@ pub(crate) fn convert_simple_data_type(sql_type: &SQLDataType) -> Result<DataTyp
         SQLDataType::Nvarchar(_)
         | SQLDataType::Uuid
         | SQLDataType::Binary(_)
+        | SQLDataType::BigNumeric(_)
+        | SQLDataType::BigDecimal(_)
         | SQLDataType::Varbinary(_)
         | SQLDataType::Blob(_)
         | SQLDataType::Datetime(_)
         | SQLDataType::Interval
+        | SQLDataType::JSON
```

Comment:
Ooh, this looks exciting. Is there some JSON routine support coming to DF?

Reply:
I think it's only supported in `sqlparser` for now.